### ISSUE TITLE
GUI: regression QR-code scanner hidden in hamburger menu

### ISSUE THREAD

**Issue by yuv** [helloworld1/FreeOTPPlus#64]
assuming this is really a fork of FreeOTP (and not a complete GUI redesign): can you please bring back from the hamburger menu on the right the two functions of ScanQR Code and Add Token to the icons they had on the older FreeOTP GUI?

Reason: these two functions are now one further tap away, with no benefit: No space was saved in putting these two function inside the hamburger menu.

**Comment by helloworld1** [comment]
Yes. it is a regression of using the material theme. I know that scan is used frequently. But is the "Add token" also used a lot? I feel probably not.

**Comment by yuv** [comment]
Does not matter since there is no benefit removing "Add token". I do not understand why "material theme" is being cited as the reason for the regression?
